src/main/java/frc/team2412/robot/util/InterpolatingTreeMap.java

```diff
@@ -0,0 +1,67 @@
+package frc.team2412.robot.util;
+
+import java.util.TreeMap;
+
+public class InterpolatingTreeMap extends TreeMap<Double, ShooterDataDistancePoint> {
+    // TODO: Use JSON files in deploy folder?
+
+    public InterpolatingTreeMap() {
+        super();
+    }
+
+    public InterpolatingTreeMap(ShooterDataDistancePoint[] dataPoints) {
```

Comment:
Something to consider for future consideration if we don't get time to properly tune the shooter.
We should talk to the drive team about possibly adding offsets they can update via shuffleboard during the match if shooter is consistently long, short, or off center.